mink/lie/se3.py

```diff
@@ -215,6 +217,58 @@ def ljacinv(cls, other: np.ndarray) -> np.ndarray:
         return np.block([[J_inv, -J_inv @ Q @ J_inv], [O, J_inv]])
 
 
+def interpolate_se3(
+    t_from: SE3, t_to: SE3, lin_threshold: float, ori_threshold: float
+) -> List[SE3]:
+    """Interpolate two transforms via decoupled translation and rotation.
+
+    Args:
+        t_from: The interpolation start transform.
+        t_to: The interpolation end transform.
+        lin_threshold: Maximum linear distance allowed between adjacent transforms.
+        ori_threshold: Maximum orientation distance allowed between adjacent transforms.
```

Comment:
Is it safe to assume we are always dealing with radians for orientation? If so, I can add docs that mention radians as the unit for `ori_threshold`. Also, for `lin_threshold`, are there units I should specify?

Reply:
marking this as resolved since we are moving over to an internal implementation as discussed below